tests/baselines/reference/docker/vue-next.log

```diff
@@ -34,93 +44,110 @@ The API report is up to date: temp/runtime-test.api.md
-Warning: /vue-next/packages/runtime-core/src/component.ts:322:19 - (TS7006) Parameter 'err' implicitly has an 'any' type.
 /vue-next/packages/server-renderer/src/index.ts → packages/server-renderer/dist/server-renderer.cjs.js...
-(!) Generated an empty bundle
 created packages/server-renderer/dist/server-renderer.cjs.js in ?s
 /vue-next/packages/server-renderer/src/index.ts → packages/server-renderer/dist/server-renderer.cjs.prod.js...
-(!) Generated an empty bundle
 created packages/server-renderer/dist/server-renderer.cjs.prod.js in ?s
-(node:18) UnhandledPromiseRejectionWarning: Error: Internal Error: Unable to determine module for: /vue-next/packages/server-renderer/dist/packages/server-renderer/src/index.d.ts
-You have encountered a software defect. Please consider reporting the issue to the maintainers of this application.
-    at ExportAnalyzer._getModuleSymbolFromSourceFile (/vue-next/node_modules/@microsoft/api-extractor/lib/analyzer/ExportAnalyzer.js:146:15)
-    at ExportAnalyzer.fetchAstModuleFromSourceFile (/vue-next/node_modules/@microsoft/api-extractor/lib/analyzer/ExportAnalyzer.js:41:35)
-    at AstSymbolTable.fetchAstModuleFromWorkingPackage (/vue-next/node_modules/@microsoft/api-extractor/lib/analyzer/AstSymbolTable.js:53:37)
-    at Collector.analyze (/vue-next/node_modules/@microsoft/api-extractor/lib/collector/Collector.js:114:51)
-    at Function.invoke (/vue-next/node_modules/@microsoft/api-extractor/lib/api/Extractor.js:97:19)
-    at build (/vue-next/scripts/build.js:91:30)
-    at processTicksAndRejections (internal/process/task_queues.js:89:5)
-    at async buildAll (/vue-next/scripts/build.js:45:5)
-    at async /vue-next/scripts/build.js:35:5
-(node:18) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
-(node:18) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
+/vue-next/packages/size-check/src/index.ts → packages/size-check/dist/size-check.global.js...
+created packages/size-check/dist/size-check.global.js in ?s
+/vue-next/packages/size-check/src/index.ts → packages/size-check/dist/size-check.global.prod.js...
+created packages/size-check/dist/size-check.global.prod.js in ?s
+/vue-next/packages/template-explorer/src/index.ts → packages/template-explorer/dist/template-explorer.global.js...
+created packages/template-explorer/dist/template-explorer.global.js in ?s
+/vue-next/packages/vue/src/index.ts → packages/vue/dist/vue.esm-bundler.js...
+[!] (plugin rpt2) Error: /vue-next/packages/vue/src/devCheck.ts(2,11): semantic error TS2774: This condition will always return true since the function is always defined. Did you mean to call it instead?
```

Comment:
[this](https://github.com/vuejs/vue-next/blob/master/packages/vue/src/devCheck.ts#L2) is due to us expending the uncalled functions checks to ternaries. As far as I'm concerned, the error's not wrong (this is a defensive check).